grass7/imagery/i.sentinel/i.sentinel.import/i.sentinel.import.py

```diff
@@ -491,7 +499,32 @@ def create_register_file(self, filename):
                         br='S2_{}'.format(band_ref)
                     ))
                 fd.write(os.linesep)
+
+    def set_footprint(self,fp):
```

Comment:
If you use: `gs.use_temp_region()` (see: https://grass.osgeo.org/grass78/manuals/libpython/script.html?highlight=use_temp#script.core.use_temp_region) it is not necessary to capture the current region and it is also safer as an atexit handler is Integrated into "use_temp_region".

Reply:
now included (see above)